src/libusermode/printers/printers.cpp

```diff
@@ -341,8 +341,8 @@ std::string GuidPrinter::print(drakvuf_t drakvuf, drakvuf_trap_info* info, uint6
     return name + "=" + std::string(stream);
 }
 
-BitMaskPrinter::BitMaskPrinter(std::string arg_name, bool print_no_addr, std::map < uint64_t, std::string > dict)
-    : ArgumentPrinter(arg_name, print_no_addr)
+BitMaskPrinter::BitMaskPrinter(const PrinterConfig& config, std::string arg_name, std::map < uint64_t, std::string > dict)
+    : ArgumentPrinter(config, arg_name)
```

Comment:
Well, if we're reordering arguments please keep same order between similar structs.